console/src/main/webapp/manager/app/assets/lang/fr.json

```diff
@@ -109,6 +109,7 @@
   "org.description"       : "Description",
   "org.url"               : "Site web",
   "org.logo"              : "Logo",
+  "org.note"              : "Notes",
```

Comment:
Missing NL translation

Reply:
on a sidenote nl.json lacks at least org.description, org.url, org.logo, org.cities, org.orgType & imageinput.label...

Reply:
Indeed.
Do you want me to reformat it ?

Reply:
i'll merge/add the missing keys when adding the new ones..

Reply:
Done with 0d0405374a99167d5aafcbcddc91758158858138

Reply:
ping @bendeg

Reply:
c7ab0bb should take care of most of the missing ones, of course to be translated from en to nl.